src/app/components/ErrorPage/__snapshots__/index.test.jsx.snap

```diff
@@ -293,11 +292,10 @@ exports[`ErrorMain should correctly render for an error page for Persian 1`] = `
   font-size: 2rem;
   line-height: 2.625rem;
   color: #3F3F42;
-  font-family: NassimPersian,Arial,Verdana,Geneva,Helvetica,sans-serif;
-  font-weight: 700;
-  font-style: normal;
+  font-family: ReithSerif,Helvetica,Arial,sans-serif;
```

Comment:
Hmm, something is causing the persian error page to now use 'news' as the service name, not 'persian' in this test